rDNA/R/rDNA.R

```diff
@@ -5504,15 +5501,15 @@ dna_timeWindow <- function(connection,
           return(mod.m)
         })
       }
-    }
+  }
```

Comment:
Looks like the indentation is messed up in lines 5504-5512.

Reply:
Strange. I didn't touch that function. I corrected it.